talks/views.py

```diff
@@ -47,16 +47,17 @@ def talkRequest(request):
 
 def get_page(request):
     """This gets the list of objects of the next page. Assists in the pagination on scroll"""
-    topics_list = Topic.objects.values_list('talk__name', 'creator__username', 'description', 'date_created', 'views')
-    paginator = Paginator(topics_list, 4)
+
+    posts_list = Post.objects.values_list('talk__name', 'creator__username', 'content', 'date_created')
```

Comment:
Incorporate `select_related` into your code, so a `join` is performed on the `db` level, before it gets to the application